strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/login/LoginController.java

```diff
@@ -46,6 +51,9 @@
     @Inject
     private SecurityTokenProvider securityTokenProvider;
 
+    @Inject
+    @JwtAuthenticationClaimsProvider.JwtAuthentication
```

Comment:
It'd look better if you added 
`import org.carlspring.strongbox.users.security.JwtAuthenticationClaimsProvider.JwtAuthentication;`

```
    @Inject
    @JwtAuthentication
    private JwtClaimsProvider jwtClaimsProvider;
```

this comment also relates to the the similar places in the PR

Reply:
> this comment also relates to the the similar places in the PR

Reply:
i've fixed it everywhere where possible